awx_collection/README.md

```
@@ -55,9 +44,17 @@ username = foo
 password = bar
 ```
 
+or like this:
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)